src/Entity/IdeasWorkshop/Answer.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Answer
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $text;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Adherent", inversedBy="noteAnswers")
+     */
+    private $adherent;
+
+    /**
+     * @ORM\OneToOne(targetEntity="Question", inversedBy="answer")
```

Comment:
a question can have only one answer?

Reply:
Yes, a redactor will have to write one answer per question.